openupgradelib/openupgrade.py

```diff
@@ -1704,7 +1689,6 @@ def wrap(func):
         def wrapped_function(*args, **kwargs):
```

Comment:
IMHO it's better if you do:

```py
    @functools.wraps(func)
    def wrap(func):
        @functools.wraps(func)
        def wrapped_function(*args, **kwargs):
```

Instead of calling `update_wrapper` at the end.

Reply:
@Yajo why would that be better?

Reply:
it seems more idiomatic to me. Maybe just a matter of taste.

Reply:
@Yajo I changed it according to your suggestion. But actually only need one wraps per function.